jupyterfs/extension.py

```diff
@@ -42,8 +42,9 @@ def _load_jupyter_server_extension(serverapp):
         warnings.warn(_mm_config_warning_msg)
         return
 
+    serverapp.contents_manager_class = MetaManager
```

Comment:
```suggestion
```

By setting this here, it means we cannot use the setting system to configure the CM class as e.g. a subclass of MetaManager. Normally I would expect the config file (jupyter_server_config.json) to be enough. Alternatively, we could do a
```python
if not isinstance(serverapp.contents_manager_class, MetaManager): ...
```

Reply:
I couldn't get it working with just the config file, the type was set but it wasn't actually using it unless this override was present